src/configure/configure.ts

```diff
@@ -173,37 +172,57 @@ class Orchestrator {
 
     private async getGitDetailsFromRepository(): Promise<void> {
         this.localGitRepoHelper = await LocalGitRepoHelper.GetHelperInstance(this.workspacePath);
```

Comment:
Can you elaborate

Reply:
since localGitRepoHelper is not used anywhere outside this function, I was thinking if we can instead pass it as argument to the functions using it, rather than making it a global variable. As we build on this code, if we have it as global variable I think it'll make the code more convoluted. It's your call, functionally there's nothing wrong.

Reply:
Will think about this. Will check in this version for now.